recipes/full_continual_distributed_multi_dataset.py

```diff
@@ -0,0 +1,948 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
All of the contents of this file except the `_setup_data()` function was copied from [recipes/full_finetune_distributed.py](https://github.com/pytorch/torchtune/blob/d6fbab8d02f37555d938e574c1f78e48a6a13ce3/recipes/full_finetune_distributed.py)

Here is the diff comparing the 2 files:
https://www.diffchecker.com/CjtUG31A/